.changeset/nervous-otters-sell.md

```diff
@@ -0,0 +1,7 @@
+---
+'@emotion/core': patch
```

Comment:
I would split this into 2 different changesets, one for `@emotion/styled`+`@emotion/styled-base` and the other one for `@emotion/core`. That way we could write specific changelog entries for each one.

Reply:
done, new changesets appended